app/components/header/style.scss

```diff
@@ -38,3 +38,7 @@
     border-bottom-color: govuk-colour("purple");
   }
 }
+
+#navigation {
```

Comment:
I think we can just use a govuk utility class to zero out the margin `govuk-!-margin-bottom-0` on the element or something. Also we should probably avoid targeting id selectors due to the specificity